packages/@react-aria/dnd/stories/dnd.css

```diff
@@ -185,3 +185,76 @@
 .option.drop-target {
   outline: 2px solid blue;
 }
+
+@media (forced-colors: active) {
+  .draggable {
+    --spectrum-dropzone-background-color-selected-hover: Canvas;
+    --spectrum-dropzone-border-color-selected-hover: Highlight;
+    --spectrum-alias-highlight-selected: Highlight;
+    --spectrum-table-row-text-color-selected: HighlightText;
+    --spectrum-table-row-background-color-selected: Highlight;
+
+    &:focus-ring {
+      forced-color-adjust: none;
+    }
+
+    &.is-selected {
+      forced-color-adjust: none;
+      color: var(--spectrum-table-row-text-color-selected);
+      background-color: var(--spectrum-table-row-background-color-selected);
+
+      .drag-handle {
+        --spectrum-dropzone-border-color-selected-hover: HighlightText;
+        &:focus-ring {
+          forced-color-adjust: none;
+        }
+      }
+    }
+
+    &.is-dragging {
+      --spectrum-global-color-gray-75: Canvas;
+      --spectrum-global-color-gray-200: CanvasText;
```

Comment:
don't override globals

Reply:
This is a Storybook example for the dnd hooks, so I wasn't being as careful with it.

Reply:
oo, yep, those are stories, i'm sorry, i did not notice that, should be fine